pkg/speccheck/speccheck.go

```diff
@@ -110,7 +111,12 @@ func (sc *SpecCheck) compareActualVsSpec(spec, actual *unstructured.Unstructured
 	delete(updated.Object, "status")
 
 	if !equality.Semantic.DeepEqual(updated.Object, actualClone.Object) {
-		log.Printf("Objects are different: %s",
+		gk := spec.GroupVersionKind().GroupKind()
+		if gk.Group == core_v1.GroupName && gk.Kind == "Secret" {
+			log.Printf("Objects are different: Secret object %s has changed", spec.GetName())
+			return updated, false, nil
+		}
+		log.Printf("Objects are different: %q",
```

Comment:
You changed `%s` -> `%q` in the wrong place :D Please change this place back to avoid crazy quoting.